composer/trainer/devices/device_cpu.py

```diff
@@ -11,6 +13,8 @@
 
 logger = logging.getLogger(__name__)
 
+__all__ = ["DeviceCPU"]
```

Comment:
Can you denote in the init signature that this class takes no arguments? Maybe give an example of how you would pass this in when constructing the trainer.

Reply:
Discussed offline - not including an example due to simplicity of the use case + the fact that DeviceGPU can't have one due to not being able to run doctest with GPUs